src/gc/proxy.d

```diff
@@ -307,6 +311,13 @@ extern (C)
         return proxy.gc_runFinalizers( segment );
     }
 
+    bool gc_inFinalizer() nothrow
+    {
+        if( proxy is null )
```

Comment:
All code you added uses `if (proxy is null)` (i.e. different spacing), please fix this one